documentation/source/building.rst

```diff
@@ -373,6 +373,14 @@ The following variables are makefile variables, not environment variables.
 
       Use to add additional dependencies to the simulation target.
 
+.. make:var:: COCOTB_NVC_TRACE
+
+      Set this to 1 to enable display of VHPI traces when using the NVC VHDL simulator.
+
+      .. deprecated:: 1.4
+
+         Replaced by the instructions in :ref:`sim-nvc`.
+
```

Comment:
Since this PR is not a 100% revert anyway, we don't need to reintroduce this IMO.

```suggestion
```